hey @bestbeforetoday @atoulme I have fixed the compilation problem, however now it fails with the following:

```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  19:49 min
[INFO] Finished at: 2022-03-24T13:50:24Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.owasp:dependency-check-maven:6.5.3:check (default) on project fabric-sdk-java: 
[ERROR] 
[ERROR] One or more dependencies were identified with vulnerabilities that have a CVSS score greater than or equal to '7.0': 
[ERROR] 
[ERROR] kotlin-stdlib-common-1.4.0.jar: CVE-2020-15824
[ERROR] 
[ERROR] See the dependency-check report for more details.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
##[error]Bash exited with code '1'.
```
However I can't see how this is related to my changes. 
Any ideas what is going on here?

--- a/pom.xml
+++ b/pom.xml
@@ -81,14 +81,14 @@
             <dependency>
                 <groupId>io.opentelemetry</groupId>
                 <artifactId>opentelemetry-bom</artifactId>
-                <version>1.9.1</version>
+                <version>1.12.0</version>
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
             <dependency>
                 <groupId>io.opentelemetry.instrumentation</groupId>
                 <artifactId>opentelemetry-instrumentation-bom-alpha</artifactId>
-                <version>1.9.2-alpha</version>
+                <version>1.12.0-alpha</version>
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>

--- a/src/main/java/org/hyperledger/fabric/sdk/helper/Config.java
+++ b/src/main/java/org/hyperledger/fabric/sdk/helper/Config.java
@@ -26,7 +26,7 @@ import java.util.concurrent.atomic.AtomicLong;
 import io.opentelemetry.api.OpenTelemetry;
 import io.opentelemetry.sdk.OpenTelemetrySdk;
 import io.opentelemetry.sdk.OpenTelemetrySdkBuilder;
-import io.opentelemetry.sdk.autoconfigure.OpenTelemetrySdkAutoConfiguration;
+import io.opentelemetry.sdk.autoconfigure.AutoConfiguredOpenTelemetrySdk;
 import io.opentelemetry.sdk.trace.SdkTracerProvider;
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
@@ -247,7 +247,11 @@ public class Config {
 
             }
 
-            openTelemetry = OpenTelemetrySdkAutoConfiguration.initialize(false);
+            openTelemetry = AutoConfiguredOpenTelemetrySdk
+                .builder()
+                .setResultAsGlobal(false)
+                .build()
+                .getOpenTelemetrySdk();
         }
 
     }

